pkg/trace/trace.go

```diff
@@ -341,11 +341,3 @@ func FromHeaders(ctx context.Context, hdrs map[string][]string, name string) con
 	}
 	return defaultTracer.fromHeaders(ctx, hdrs, name)
 }
-
-// ForceFlush immediately exports all the spans that have been buffered to this point
```

Comment:
Breaking change?

Reply:
Actually, no. This change was introduced in the initial Delibird migration (not merged to main yet). I realized that https://github.com/getoutreach/gobox/pull/240/files#diff-5b8f2be4e453643719aa5e585f8e8a1d73bfb8b873c1ac8d3902279f1b2e90f5R50 will force a flush for us.